internal/nodes/avi_model_nodes.go

```diff
@@ -761,11 +761,7 @@ func (v *AviVSVIPNode) GetCheckSum() uint32 {
 
 func (v *AviVSVIPNode) CalculateCheckSum() {
 	var checksum uint32
-	if v.NetworkName != nil {
-		checksum = lib.VSVipChecksum(v.FQDNs, v.IPAddress, *v.NetworkName)
-	} else {
-		checksum = lib.VSVipChecksum(v.FQDNs, v.IPAddress, "")
-	}
+	checksum = lib.VSVipChecksum(v.FQDNs, v.IPAddress, v.NetworkNames)
```

Comment:
if you simply do this, then the checksum will change even if the NetworkNames slice is empty and this will re-sync all vips. Note: Empty slices does not mean a 0 checksum.